zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -30,13 +32,22 @@
     PROFILE_ID,
 )
 from zhaquirks.xiaomi import (
+    AQARA,
     LUMI,
     BasicCluster,
     XiaomiAqaraE1Cluster,
     XiaomiCustomDevice,
 )
 
 
+class LumiPowerOnBehaviorMode(t.enum8):
+    """Power on behavior mode."""
+
+    On = 0x00
+    PreserveState = 0x01
```

Comment:
Can we use `LastState` here? That's similar to some config entities still in ZHA for Tuya plugs.
```suggestion
    LastState = 0x01
```